network-api/networkapi/wagtailpages/pagemodels/blog/blog.py

```diff
@@ -117,11 +93,11 @@ def get_context(self, request):
         context['coral_talk'] = context['coral_talk_server_url'] and self.feature_comments
 
         # Pull this object specifically using the English page title
-        blog_page = IndexPage.objects.get(title_en__iexact='Blog')
+        blog_page = BlogIndexPage.objects.get(title_en__iexact='Blog')
```

Comment:
fun fact: this change means everything else had to be done, because of how both BlogPage and BlogIndexPage rely on the category...